_t/env.sh

```diff
@@ -1,4 +1,8 @@
 export COLUMNS=149
+if command -v stty 1>/dev/null 2>/dev/null
+then
+  stty columns 149
```

Comment:
This will change my tty params even after the tests exit.  I think we need to preserve/restore the original value to make this sane.  Something like:
```
if command -v stty 1>/dev/null 2>/dev/null
then
    cur=$(stty size  | cut -d " " -f 2)
    stty columns 149
    stty size
    restoreCols() {
        stty columns $cur
        stty size
    }
    trap restoreCols INT TERM EXIT
fi
```

Reply:
That makes sense, though I can't shake the feeling that there's a "right" way to set up a fake TTY for tests like this. I'll do some research when I get a minute tomorrow.

Reply:
although it looks like #302 obviates the need for this PR; I'll double-check tomorrow and close if so.